src/api/store/event.py

```diff
@@ -257,6 +253,8 @@ def create_event(self, context: Context, args) -> Tuple[dict, MassEnergizeAPIErr
         if not community:
           return None, CustomMassenergizeError("Please provide a valid community_id")
 
+      print("before event .create()")
```

Comment:
Not super important, but these print debugs should be removed when this PR gets merged.

Reply:
right you are.  I've removed them